src/vs/workbench/electron-browser/window.ts

```diff
@@ -169,6 +169,9 @@ export class ElectronWindow extends Disposable {
 			}
 		});
 
+		// Support open event for opener service
```

Comment:
Desktop only.

Reply:
```suggestion
		// Support open event for opener service in desktop only
```

Yo, we can use suggested changes!